lessons/beginners/functions/index.md

```diff
@@ -289,10 +289,10 @@ Pozor na to, že když nenapíšeš závorky, funkce se nezavolá!
 Výraz `len(s)` je *volání funkce*, ale `len` bez závorek označuje
 *funkci samotnou*.
 
-Výsledek `len(s)` je číslo; `len` je funkce.
+Výsledek `len(s)` je číslo; výsledek `len` je funkce.
```

Comment:
Tahle změna mi přijde méně srozumitelná než originál.

Reply:
Co tohle: Výsledek výrazu `len(s)` je číslo; výsledek výrazu `len` je funkce `len`.

Šlo mi o to, aby tam nebylo jen 'len je funkce'. To je napsáno hned nad tím.

Reply:
...výsledek výrazu len je samotná funkce len.